carsharing/simulator/deployment.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: car-simulator
+spec:
+  selector:
+    matchLabels:
+      app: car-simulator
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: car-simulator
+    spec:
+      containers:
+        - name: car-simulator
+          image: us.gcr.io/d9p-quick/demo/car-simulator:latest
```

Comment:
Is there a new image for this?

Reply:
removed deployment, as we only use the script in the demo...